src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/RcaController.java

```diff
@@ -276,6 +276,11 @@ private void readRcaEnabledFromConf() {
    * <p>In case all the RCAs in param value are incorrect, return without any update.
    */
   private void readAndUpdateMutesRcas() {
+    if (rcaConf == null) {
```

Comment:
In what case can this be `null` ? This shouldn't ever be null. RCA system depends on this config. Scheduler etc. cannot even start when this is null.

Reply:
The [rca config is initialized](https://github.com/opendistro-for-elasticsearch/performance-analyzer-rca/blob/master/src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/RcaController.java#L146) in the `start()` as part of [`updateRcaState()`](https://github.com/opendistro-for-elasticsearch/performance-analyzer-rca/blob/master/src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/RcaController.java#L334) invocation. 

The readAndUpdateMutesRcas() is invoked from [`readRcaEnabledFromConf()`](https://github.com/opendistro-for-elasticsearch/performance-analyzer-rca/blob/master/src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/RcaController.java#L250), which happens before the `updateRcaState()`call, thus leading to NPE.